third_party/bazel/blade_gemm/blade_gemm.BUILD.tpl

```diff
@@ -0,0 +1,39 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+load("@local_config_blade_disc_helper//:build_defs.bzl",
+    "cc_bin_path",
+    "cxx_bin_path",
+)
+
+filegroup(
+    name = "all_source_files",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:private"],
+)
+
+cmake(
+    name = "blade_gemm",
+    cache_entries = {
+        "CMAKE_BUILD_TYPE": "Release",
```

Comment:
nit: if binary size is not a concern, `RelWithDebInfo` is preffered.

Reply:
libblade_gemm.a has a size of 66MB, I think it is quite big already.